telemetry/definitions/commonDefinitions.json

```diff
@@ -1000,6 +1026,16 @@
                 { "type": "failedCount", "required": false}
             ]
         },
+        {
+            "name": "aws_filetypes_edit",
```

Comment:
This is a departure from the current metric naming conventions. I don't have strong preference here, but its worth discussing before accepting this.

Style-wise, I don't know if the second underscore will break any assumptions around metric names.

Scanning the current metrics, it looks like this would be called something like `aws_editAwsFile`. I am guessing you want to scope the metric to "file related operations", which is handy if we add more file related metrics in the future. Today, the first portion of metric names roughly correlates to scope, what if we named it `file_editAwsFile`?

Reply:
> This is a departure from the current metric naming conventions. I don't have strong preference here, but its worth discussing before accepting this.

Yeah for these definitions I'd prefer not deviating from convention just yet (it makes it harder to do widespread refactoring). `file_editAwsFile` can be thought of as a metric for the 'file' subsystem which I think that's the intent?

I'm okay with a different format so long as we know how it should be interpreted.